splunk_logger.go

```diff
@@ -568,6 +568,7 @@ func telemetry(info logger.Info, l *splunkLogger, sourceType string, splunkForma
 		url:       l.hec.url,
 	}
 
+	time.Sleep(5 * time.Second)
```

Comment:
What is the sleep for?

Reply:
The sleep prevents the telemetry function from throwing off the event count for the unit tests